ui/package.json

```diff
@@ -10,7 +10,7 @@
     "formik": "1.5.1",
     "lodash.isempty": "^4.4.0",
     "lodash.sortby": "^4.7.0",
-    "oidc-client": "^1.8.0",
+    "oidc-client": "^1.9.1",
```

Comment:
why do you need to update?

Reply:
oh. ok. You used v 1.8.0. I will keep the same as well.